ckanext/geocat/utils/ogdch_map_utils.py

```diff
@@ -89,16 +89,26 @@ def map_to_ogdch_categories(geocat_categories):
 
 def map_frequency(geocat_frequency):
     frequency_mapping = {
-        'continual': 'http://purl.org/cld/freq/continuous',
-        'daily': 'http://purl.org/cld/freq/daily',
-        'weekly': 'http://purl.org/cld/freq/weekly',
-        'fortnightly': 'http://purl.org/cld/freq/biweekly',
-        'monthly': 'http://purl.org/cld/freq/monthly',
-        'quarterly': 'http://purl.org/cld/freq/quarterly',
-        'biannually': 'http://purl.org/cld/freq/semiannual',
-        'annually': 'http://purl.org/cld/freq/annual',
-        'asNeeded': 'http://purl.org/cld/freq/irregular',
-        'irregular': 'http://purl.org/cld/freq/irregular',
+        'continual':
+            'http://publications.europa.eu/resource/authority/frequency/CONT',
+        'daily':
+            'http://publications.europa.eu/resource/authority/frequency/DAILY',
+        'weekly':
+            'http://publications.europa.eu/resource/authority/frequency/WEEKLY',
+        'fortnightly':
+            'http://publications.europa.eu/resource/authority/frequency/BIWEEKLY',
+        'monthly':
+            'http://publications.europa.eu/resource/authority/frequency/MONTHLY',
+        'quarterly':
+            'http://publications.europa.eu/resource/authority/frequency/QUARTERLY',
+        'biannually':
```

Comment:
I think it is spelled `biennually`

Reply:
according to our conversation on slack, it might be up to GEOCAT, how `biennually` is spelled :) please check what this value is used for :)

Reply:
We get from geocat [geocat_frequency ,](https://github.com/opendata-swiss/ckanext-geocat/blob/dd8f8126885b46d8aa8ae7c8a21e4931363f660d/ckanext/geocat/utils/csw_mapping.py#L309), e.g., irregular, and map it to our new url in map_frequency(geocat_frequency): 'http://publications.europa.eu/resource/authority/frequency/IRREG'

Reply:
okay, sounds good, then we need to leave those values like this, closing this thread :)